switchtheme.module

```diff
@@ -1,8 +1,11 @@
-<?php
+<?php 
 
 /**
  * @file
- * Adds a block with a user theme switcher.
+ * Adds three blocks with user theme switchers
+ *  1) A dropdown list of available themes
+ *  2) An unordered list of available themes
+ *  3) A ramdon pick of available themes
```

Comment:
Our coding standards say that sentences within inline comments should have punctuation. See https://api.backdropcms.org/php-standards#comments

> Comments within functions and methods should use capitalized sentences with punctuation.

Also see: https://api.backdropcms.org/doc-standards#lists

> The API module creates nested bullet lists from lines starting with `-` in docblocks.

```suggestion
 * Adds three blocks with user theme switchers:
 * - A dropdown list of available themes.
 * - An unordered list of available themes.
 * - A random pick of available themes.
```